util/getUniqueCurrentProjects.js

```diff
@@ -0,0 +1,24 @@
+export default function getUniqueCurrentProjects(teamMembers) {
```

Comment:
Making some assumptions for app behavior here:
- If there is no client and no project specified, we don't want to include that in the select.
- If there is a client but no project, we will include that as 'Unnamed `client` Project'.